README.md

```diff
@@ -107,7 +115,7 @@ optional arguments:
                         CRLs
   --tls-ciphers TLS_CIPHERS
                         Ciphers to include. See https://www.openssl.org/docs/m
-                        an1.1.0/man1/ciphers.html for cipher list format
+                        an1.0.1/apps/ciphers.html for cipher list format
```

Comment:
It is likely a mistake.  It is here because I made fixes in the TLS usage info across graphite and Zabbix.  After I finished Zabbix I copied it over to here.  From looking through the plugins the only one with https://www.openssl.org/docs/man1.0.1/apps/ciphers.html is Zabbix.  Should I fix it there as well?

Reply:
Ok, so I am not quite sure is was ever `man1` in the first place.  The usage in the code is consistent with this change.  It is also like this in nagios.  It looks like someone changed it in the README but not in the code so when I went to update the usage info via copy-paste I overwrote it.  I will fix it